### DISCUSSION TITLE
Linux/%E4%B9%8C%E5%90%88%E4%B9%8B%E4%BB%A4

### DISCUSSION THREAD

**Discussion by giscus** [chua-n/chua-n.github.io#155] [Announcements]
# Linux/%E4%B9%8C%E5%90%88%E4%B9%8B%E4%BB%A4

短小之令 clear：清除屏幕里的所有命令 reboot：重启系统，默认只能使用 root 管理员来重启 poweroff：关闭系统，默认只有 root 管理员可关闭 last：查看系统所有的登录记录，last <option> 由于这些信息都是以日志文件的形式保存在系统中，因此黑客可以很轻易地对内容进行篡改，千成不要单纯地以该命令的输出信息来判断系统...

http://localhost:9999/Linux/%E4%B9%8C%E5%90%88%E4%B9%8B%E4%BB%A4.html

**Comment by sppkisa** [comment]
111
